contrib/sip_proxy/filters/network/source/conn_manager.h

```diff
@@ -71,7 +52,8 @@ class ProtocolOptionsConfig : public Upstream::ProtocolOptionsConfig {
 
   virtual bool sessionAffinity() const PURE;
   virtual bool registrationAffinity() const PURE;
-  virtual const std::vector<CustomizedAffinity>& customizedAffinityList() const PURE;
+  virtual envoy::extensions::filters::network::sip_proxy::v3alpha::CustomizedAffinity
+  customizedAffinity() const PURE;
```

Comment:
Is this method used anywhere? If so could it be amended to return const ref to avoid copies of `envoy::extensions::filters::network::sip_proxy::v3alpha::CustomizedAffinity`?

Reply:
Yes, changed it into const reference. Thanks for your comments.